monasca-log-transformer/README.md

```diff
@@ -0,0 +1,40 @@
+monasca-log-transformer
+-----------------------
+
+**monasca-log-transformer** image contains [Logstash][1] configuration
+to detect log's severity.
+
+Tags
+----
+
+**monasca-log-transformer** uses simple [SemVer][2] tags as follows:
+
+* `0.0.1` - `latest`
+
+Configuration
+-------------
+
+| Variable                  |   Default        | Description                        |
+|---------------------------|------------------|------------------------------------|
+| `ZOOKEEPER_URI`           | `zookeeper:2181` | An URI to Zookeeper server         |
+| `KAFKA_URI`               | `kafka:9092`     | The host and port for kafka        |
+| `KAFKA_WAIT_FOR_TOPICS`   | `log-transformed,log` | Topics to wait on at startup |
```

Comment:
no, should be log-transformed - I will check other places and make it the same everywhere else.